src/symbols.jl

```diff
@@ -112,7 +112,12 @@ function maybe_fix_path(file)
     return maybe_fixup_stdlib_path(file)
 end
 
-safe_isfile(x) = try isfile(x); catch; false end
+safe_isfile(x) =
+    try
+        isfile(x)
+    catch
+        false
+    end
```

Comment:
These line breaks should not be added.

Reply:
```
try isfile(x); catch; false end
```

is automatically rewritten across lines atm

Reply:
Yeah, I think I probably identified a couple of instances that are all based on the same underlying logic here. We just need an option for that, right?